src/utils.ts

```diff
@@ -29,3 +30,16 @@ export const createdRandomUID = (): string => {
   //in the python version
   return uuidv4().replace(/-/g, "");
 };
+
+export const verifyJWTRequest = (req: VercelRequest) => {
+  try {
+    const authHeader = req.headers.authorization;
+    const token = authHeader && authHeader.split(" ")[1];
```

Comment:
Did you try adding jwt to your auth header and checking if this reads the jwt correctly?

Reply:
I tested it in a separate route to see if the validation worked and it did, just didn't add it as middleware though for each route, since I didn't know much about that